react/getting_started_with_react/keys_in_react.md

```diff
@@ -1,8 +1,8 @@
 ### Introduction
 
-In this lesson we will cover keys in React. Keys are special props for our components and we'll learn why they are used.
+In this lesson, we will cover keys in React. Keys are special props for our components and we'll learn why they are used.
 
-### Lesson overview
+### Lesson Overview
```

Comment:
We use "Sentence case" as per the [Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#headings).

```suggestion
### Lesson overview
```

Reply:
Hi, I made sure to check previous lessons on this but I've found that all other lessons use `Lesson Overview`, even in JS courses. 
![image](https://github.com/TheOdinProject/curriculum/assets/85281165/cc66ad77-139d-4c24-bb55-e8d7f9dbf520)

Reply:
you're absolutely right and the sentence case is a recent change, hence why we try to catch these on PR's. Would you be willing to change the headers in this lesson?

Reply:
I didn't know it was a recent change. Yes, I'll change all the headers in this lesson.